terraform/readme.md

```diff
@@ -0,0 +1,17 @@
+############################################################################################
+input steps:
+1. create a ECR repository and add the image url with tag under image section of container definiton file
```

Comment:
Codacy has a fix for the issue: [Lists should be surrounded by blank lines](https://app.codacy.com/gh/opstree/spring3hibernate/pullRequest?prid=8912394)
```suggestion

1. create a ECR repository and add the image url with tag under image section of container definiton file
```